src/index.tsx

```diff
@@ -62,14 +62,7 @@ function App() {
 
   const [justClicked, setJustClicked] = useState<boolean>(false);
 
-  // Expose DeckGL instance on window for Playwright e2e tests
   const deckRef = useRef<DeckGLRef | null>(null);
-  useEffect(() => {
-    if (deckRef.current && typeof window !== "undefined") {
-      (window as unknown as Record<string, unknown>).__deck =
-        deckRef.current.deck;
-    }
-  }, [deckRef.current]);
```

Comment:
I wonder whether it could make sense to keep this? It can be useful to access the `deck` object on the window when debugging

Reply:
I restored this for merging, but happy to continue discussion